sphinx/writers/latex.py

```diff
@@ -1161,7 +1161,9 @@ def visit_paragraph(self, node: Element) -> None:
             # (first one is label node)
             pass
         else:
-            self.body.append('\n')
+            # the \sphinxAtStartPar is to allow hyphenation of first word of
+            # a paragraph in narrow contexts such as in a table cell
+            self.body.append('\n\\sphinxAtStartPar\n')
```

Comment:
The children of paragraph node are regularly text nodes or inline nodes. It is defined at the doctree specification.
https://docutils.sourceforge.io/docs/ref/doctree.html

Almost of the inline nodes are decorated text. But, some of them are not; for example, math, image, problematic and raw. Additionally, some extensions might generate broken doctree under the paragraph node.

Reply:
A math node is fine (via the `math` role), an image (via a substitution) also is no problem. As per ``raw latex``, also inserted by a substitution then if the raw latex starts some list environment it will be shifted one line down, but it seems then a user error to do this, the raw directive should have been used directly to insert the LaTeX "display" material in place of "inline" material. I can not think of a natural example causing a problem, after all docutils "paragraph" concept pretty well matches with latex "paragraph" concept I hope.

Reply:
Agreed. From my perspective, docutils' paragraph model and LaTeX's horizontal mode are similar. (But I did not mention that yesterday because I'm not familiar with LaTeX's mode :-p)